src/pages/resultsView/cnSegments/CNSegments.tsx

```diff
@@ -7,7 +7,7 @@ import {Nav, NavItem} from "react-bootstrap";
 
```

Comment:
We have another `CNSegments` tab for study view: `src/pages/studyView/tabs/CNSegments.tsx`. We may want to update that one as well with the genome information.

Reply:
**_CNSegments.tsx_** page is not affected, since **_IntegrativeGenomicsViewer_** is the one load genome reference fa file from s3.

Reply:
should IGV be given some parameter to pick the correct reference genome?